service/udp.go

```diff
@@ -21,7 +21,8 @@ import (
 	"runtime/debug"
 	"time"
 
-	"github.com/Jigsaw-Code/outline-ss-server/metrics"
+	"github.com/Jigsaw-Code/outline-ss-server/service/metrics"
```

Comment:
Nit: This import order and spacing seems extremely arbitrary.

Reply:
Updated